samza-kv/src/main/scala/org/apache/samza/storage/kv/KeyValueStorageEngine.scala

```diff
@@ -52,8 +52,11 @@ class KeyValueStorageEngine[K, V](
   }
 
   override def getAll(keys: java.util.List[K]): java.util.Map[K, V] = {
-    metrics.gets.inc(keys.size)
-    wrapperStore.getAll(keys)
+    updateTimer(metrics.getAllNs) {
```

Comment:
Does it makes sense to add metric assertions for new counters to existing `TestKeyValueStorageEngine` test

Reply:
There are no existing tests for getAll/putAll etc. Since this change doesn't change the existing logic, and IMHO metrics update isn't something worth testing by itself, I'd prefer to skip those for now.